ibc-core/ics24-host/types/src/path.rs

```diff
@@ -22,8 +22,13 @@ const UPGRADED_CLIENT_CONSENSUS_STATE: &str = "upgradedConsState";
 /// The Path enum abstracts out the different sub-paths.
 #[derive(Clone, Debug, PartialEq, Eq, PartialOrd, Ord, Hash, From, Display)]
 pub enum Path {
+    NextClientSequence(NextClientSequencePath),
+    NextConnectionSequence(NextConnectionSequencePath),
+    NextChannelSequence(NextChannelSequencePath),
     ClientState(ClientStatePath),
     ClientConsensusState(ClientConsensusStatePath),
+    ClientConsensusStateProcessedTime(ClientConsensusStateProcessedTimePath),
+    ClientConsensusStateProcessedHeight(ClientConsensusStateProcessedHeightPath),
```

Comment:
For the alignment with ibc-go, let's append the [`processedTime`](https://github.com/cosmos/ibc-go/blob/016a6095b577ecb9323edad508cff19d017636a1/modules/light-clients/07-tendermint/store.go#L39-L40) and [`processedHeight`](https://github.com/cosmos/ibc-go/blob/016a6095b577ecb9323edad508cff19d017636a1/modules/light-clients/07-tendermint/store.go#L41-L42) to the `ClientConsensusStatePath`.
Also, I suggest to use shorter names for the variants, such as `ClientProcessedTimePath` and `ClientProcessedHeightPath`

Reply:
Forgot to document why I chose the format of the new paths. If we append `/processedTime` at the end of `ClientConsensusStatePath` - the `.keys()` call on it will return these paths too. I wanted to keep the _buckets_ separate - as it's easier to iterate when we call `.keys()`.

I agree with the shorter names.

Reply:
> Forgot to document why I chose the format of the new paths. If we append /processedTime at the end of ClientConsensusStatePath - the .keys() call on it will return these paths

I'm still inclined to go with ibc-go's data structure. It's all about keeping data structure accurate at the type level to avoid breaking changes later on. Even though ibc-go's approach might not be the most efficient for queries, it makes `ConsensState` and its metadata linked. Can you take another look and see if we can handle `get_keys` differently? What's your take on this?